packages/katello/katello/hostname-change.rb

```diff
@@ -140,18 +140,15 @@ def next_steps_message
       else
         # the following multi-line string isn't indented because the indents are taken literally.
 %(
-  You will have to install the new bootstrap rpm and reregister all clients and #{@plural_proxy} with subscription-manager
-  (update organization and environment arguments appropriately):
+  You will have to reregister all clients and #{@plural_proxy} with the Satellite server.
+  The global registration method requires an activation key and root privileges on the client server.
 
-  yum remove -y katello-ca-consumer*
-  rpm -Uvh http://#{@new_hostname}/pub/katello-ca-consumer-latest.noarch.rpm
-  subscription-manager register --org="Default_Organization" --environment="Library" --force
-
-  Then reattach subscriptions to the client(s) and run:
-
-  subscription-manager refresh
-  yum repolist
+  For each client and #{@plural_proxy}, go to Hosts > Register Host in the Satellite UI.
```

Comment:
I think this is a poor experience. Users can't script this and if you have 100s or 1000s of hosts it's not going to be acceptable.

IMHO we should have a remote execution job that updates the registration instead. Then we can tell users it can be called via Hammer as well.

Reply:
I'm not the expert here, so I would like to see if there's already other ways to achieve what you're talking about. If not, then we should spend time creating a solution like you described.